index.bs

```diff
@@ -12,72 +12,16 @@ Complain About: accidental-2119 no
 
 <pre class="link-defaults">
 spec: infra; type: dfn; text: list
-spec: url; type: interface; text: URL
-spec: dom; type: interface; text: Document
 spec: ecma-262; type: dfn; for: /; text: internal method
 spec: ecma-262; type: dfn; for: /; text: internal slot
 spec: ecma-262; type: dfn; for: /; text:realm
+spec:wasm-js-api-2;
```

Comment:
No variant without a version? cc @Ms2ger

Reply:
Yeah, it'll be fixed separately, but in the meantime this link-default fixes the error locally.